build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
They're different repositories, so I would just leave both as a safeguard, but I'm sure that we can get by using just jcenter() so I see where you're coming from.